build.sh

```diff
@@ -107,10 +110,18 @@ buildAndPublishDivoc()
     rm -rf "$SRC_CODE"
 }
 
+updateConfigs()
+{
+    cd "$SRC_CODE/default-configuration/etcd"
+    bash updateConfigs.sh $etcd
```

Comment:
can we directly run bash "$SRC_CODE"/default-configuration/etcd/updateConfigs.sh $etcd ? This will ensure that we dont have to keep changing directories.

Reply:
the problem is the file name is key for etcd
and if we run from outside, then it throughs error as the files eg DDCC_TEMPLATE.template not found.
so in the script we have to do cd, instead we did it here